.github/workflows/ci.yml

```diff
@@ -1,45 +1,15 @@
-name: ci
-'on':
+name: CI
```

Comment:
The CI badge in the readme may need to be updated if changing from lower to uppercase, but i prefer the uppercase to be honest!

Reply:
Good spot: for consistency in the action menu, I would keep the old `ci` name.

I must say that I like the upper case too 😖